packages/@lwc/engine-server/src/__tests__/fixtures/known-boolean-attributes/default-def-html-attributes/dynamic/error.txt

```diff
@@ -1 +0,0 @@
-Cannot convert a Symbol value to a number
```

Comment:
Wait why was it trying to convert to a number and not a string?

Reply:
I'm assuming because of `tabindex`.